knownhosts_test.go

```diff
@@ -151,6 +221,7 @@ func writeTestKnownHosts(t *testing.T) string {
 		"only-ed25519.example.test:22": {generagePubKeyEd25519(t)},
 		"multi.example.test:2233":      {generatePubKeyRSA(t), generatePubKeyECDSA(t), generagePubKeyEd25519(t)},
 		"192.168.1.102:2222":           {generatePubKeyECDSA(t), generagePubKeyEd25519(t)},
+		"[fe80::abc:abc:abcd:abcd]:22": {generagePubKeyEd25519(t), generatePubKeyRSA(t)},
```

Comment:
Whoops, just noticed my function name typo ("generage" instead of "generate"), I'll fix that in a commit momentarily